src/components/HeaderWithDetails/HeaderWithDetails.tsx

```diff
@@ -5,14 +5,14 @@ import './HeaderWithDetails.scss';
 
 interface Props {
     movie: Movie;
-    handleBackToSearchClick: React.MouseEventHandler;
+    onBackToSearchClick: any;
```

Comment:
That's probably something like `(SyntheticEvent) => void`, isn't it?

Reply:
almost all callbacks oi props this this actions, I  have renamed 'on' to 'handle' and have added some types into params